teamworkApp/lib/generateData.py

```diff
@@ -0,0 +1,41 @@
+# resources:
+
+# muddersOnRails()
+# Maeve Murphy October 12, 2017
+# Last updated: 10-13-2017
```

Comment:
I like the header, maybe we should consider adding them to our other files.

Reply:
Cool, I will transfer this to my other files and everyone can feel free to do the same with theirs as they see fit!